packages/utilities/src/DelayedRender.tsx

```diff
@@ -7,7 +7,7 @@ import { getWindow } from './dom/getWindow';
  * @public
  */
 // eslint-disable-next-line deprecation/deprecation
-export interface IDelayedRenderProps extends React.Props<{}> {
+export interface IDelayedRenderProps extends React.PropsWithChildren<React.PropsWithRef<{}>> {
```

Comment:
This will remove the `key` property, based on React's comment we should never need it.
![image](https://user-images.githubusercontent.com/6677963/218390538-a597c2a5-da4a-4521-88d9-ab79d218941a.png)

Reply:
The PR you referenced uses `IReactProps` to replace `React.Props`. Why not stick with that over `React.PropsWithChildren`?

Reply:
@spmonahan `IReactProps` is defined in the React package and this is a more common package that is referenced by the React package, so I think letting this package reference React package is not a good practice.

If we want to reuse `IReactProps`, another option is to move IReactProps out from React package to a common package like this Utilities.

Please let me know which approach you think is better and I will update

Reply:
In that case let's move `IReactProps` to utilities. For compatibility it's best to keep the `key` prop imo, so I'd prefer we go with `IReactProps` in this case as well.